test/metabase/models/login_history_test.clj

```diff
@@ -34,51 +35,61 @@
+                                                           (into {} (for [ip-address ip-addresses]
+                                                                      [ip-address
+                                                                       {:description "San Francisco, California, United States"
+                                                                        :timezone    (t/zone-id "America/Los_Angeles")}])))]
+              (mt/with-temp* [LoginHistory [_ {:user_id   user-id
+                                               :device_id (str (java.util.UUID/randomUUID))}]
+                              LoginHistory [_ {:user_id   user-id
+                                               :device_id device
+                                               :timestamp #t "2021-04-02T15:52:00-07:00[US/Pacific]"}]]
 
-            (testing "don't send email on subsequent login from same device"
-              (mt/reset-inbox!)
-              (mt/with-temp LoginHistory [_ {:user_id user-id, :device_id device}]
-                (is (= {}
-                       @mt/inbox)))))))))
+                (is (schema= {(s/eq email)
+                              [{:from    su/Email
+                                :to      (s/eq [email])
+                                :subject (s/eq (format "We've Noticed a New Metabase Login, %s" first-name))
+                                :body    [(s/one {:type    (s/eq "text/html; charset=utf-8")
+                                                  :content s/Str}
+                                                 "HTML body")]}]}
+                             @mt/inbox))
+                (let [message (-> @mt/inbox (get email) first :body first :content)]
+                  (testing (format "\nMessage = %s" (pr-str message))
+                    (is (string? message))
+                    (when (string? message)
+                      (doseq [expected-str ["We've noticed a new login on your Metabase account."
+                                            "We noticed a login on your Metabase account from a new device."
+                                            "Browser (Chrome/Windows) - San Francisco, California, United States"
+                                            "April 2, 2021 3:52:00 PM (Pacific Daylight Time)"]]
+                        (is (str/includes? message expected-str))))))
+
+                (testing "don't send email on subsequent login from same device"
+                  (mt/reset-inbox!)
+                  (mt/with-temp LoginHistory [_ {:user_id user-id, :device_id device}]
+                    (is (= {}
+                           @mt/inbox)))))))))))
 
   (testing "don't send email if the setting is disabled by setting MB_SEND_EMAIL_ON_FIRST_LOGIN_FROM_NEW_DEVICE=FALSE"
     (mt/with-temp User [{user-id :id, email :email, first-name :first_name}]
-      (testing "send email on first login from new device"
```

Comment:
I just removed the `testing` context here because it was just copypasta from the test above and shouldn't have been here -- actual test itself it unchanged